pipelines/ci-sit-kubernetes-pipeline.yml

```diff
@@ -164,17 +165,16 @@ jobs:
             curl -L https://git.io/get_helm.sh | bash && \
             helm init --client-only && \
             helm plugin install https://github.com/rimusz/helm-tiller
-            ./apply.sh
+            WFH_IP="35.246.106.81/32" ./apply.sh
```

Comment:
im assuming this is the concourse IP? maybe we should source this from somewhere as opposed to having it directly here....

Reply:
Good point, I could raise a PR in the terraform to add it as config and whitelist it, rather than leaving this here abusing the WFH_IP option.

Reply:
Actually this was only necessary to run in sanbox concourse so I've just removed it all together.